ledger/src/shred.rs

```diff
@@ -592,17 +594,11 @@ impl From<ShredCode> for Shred {
     }
 }
 
-impl From<ShredData> for Shred {
-    fn from(shred: ShredData) -> Self {
-        Self::ShredData(shred)
-    }
-}
```

Comment:
If we decide not to remove the `impl From<ShredCode> for Shred` block above (per my other comment) in this PR, I say we keep this block + use it to keep the lines in my other comments consistent